ui/k-info-panel/main.kytos

```diff
@@ -0,0 +1,307 @@
+<template>
+    <div v-if="render_value > 0">
+        <div class="maintenance_container">
+            <div class="maintenance-table no-compact">
+                <div class="maintenance-table-divisor">
+                    <table id="maintenance-table-list-windows">
+                        <thead>
+                            <tr>
+                                <template v-for="(header, index) in table_headers">
+                                    <th scope="col" class="header" @click="sort(index)">
+                                        {{ header }}
+                                        <span class="order-dir" v-if="current_sort === index">
+                                            {{ spanDir(index) }}
+                                        </span>
+                                    </th>
+                                </template>
+                            </tr>
+                            <tr class="header-filter">
+                                <template v-for="header in table_headers_id">
+                                    <th scope="col">
+                                        <input :id="header + '_search'" @keyup="filterWindowList"></input>
+                                    </th>
+                                </template>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            <tr v-for="row in rowsOfPage">
+                                <template v-for="column in row">
+                                    <td>{{column}}</td>
+                                </template>
+                            </tr>
+                        </tbody>
+                    </table>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div v-else>
+        <div class="maintenance_container">
+            <p class='empty-window-list'>No maintenance windows available.<p>
+        </div>
+    </div>
+</template>
+
+<script>
+ module.exports = {
+   data () {
+     return {
+       table_headers: ['id', 'start', 'end', 'description', 'items', 'status'],
+       table_headers_id: ['maintenance_id', 'maintenance_start', 'maintenance_end', 'maintenance_description', 'maintenance_items', 'maintenance_status'],
+       current_sort: 0,
+       current_sort_dir: [],
+       table_rows: [],
+       render_value: 0
+     }
+   },
+   methods: {
+     /*
+         Set the sort direction for the given column index.
+     */
+     sort: function(new_sort) {
```

Comment:
Regarding sorting, could we have by the default the `start` column sorted as ascending by default instead of `id`? I'm thinking that way, out of the box, it might be easier for operators to follow the maintenance windows especially when there are several of them.